src/IdentityManager2/Api/Controllers/PageController.cs

```diff
@@ -29,7 +29,7 @@ public async Task<IActionResult> Index()
             return View("/Areas/IdentityManager/Pages/Index.cshtml", new PageModel
             {
                 PathBase = Request.PathBase,
-                Model = JsonConvert.SerializeObject(new
+                Model = JsonSerializer.Serialize(new PageModelParams
```

Comment:
some real class implementation is needed for System.TExt.JSon to work, no annonymous types